Dockerfile

```diff
@@ -37,8 +37,6 @@ RUN apt-get update \
   && mkdir -p /opt/pandoc/templates && tar zxf ${PANDOC_TEMPLATES_VERSION}.tar.gz \
   && cp -r pandoc-templates*/* /opt/pandoc/templates && rm -rf pandoc-templates* \
   && mkdir /root/.pandoc && ln -s /opt/pandoc/templates /root/.pandoc/templates \
-  && apt-get clean \
```

Comment:
Moved this in a separate clean-up step that we run only after all apt packages are installed.